firstvoices/backend/views/search/base_search_views.py

```diff
@@ -79,4 +87,11 @@ def list(self, request, **kwargs):
         # Adding data to objects
         hydrated_objects = hydrate_objects(search_results, request)
 
+        page = self.paginate_queryset(hydrated_objects)
+        # TODO: Use dictionary entry serializer to properly serialize search results
```

Comment:
Currently pagination is returning all hydrated objects on multiple pages, rather than splitting them into the multiples pages as expected. Once the hydrated objects are properly serialized (needs camelcase to underscore conversion) pagination should function as expected)